src/main/java/at/hannibal2/skyhanni/features/combat/endernodetracker/EnderNodeTracker.kt

```diff
@@ -184,7 +183,9 @@ object EnderNodeTracker {
         return newProfit
     }
 
-    private fun isInTheEnd() = LorenzUtils.skyBlockArea == "The End"
+    private fun isEnabled() =
+        LorenzUtils.inSkyBlock && IslandType.THE_END.isInIsland() && config.enabled &&
+            (!config.onlyPickaxe || ItemCategory.miningTools.containsItem(InventoryUtils.getItemInHand()))
```

Comment:
Maybe extract into function "hasItemInHand()" for readability

Reply:
I was thinking of this, but It's fine.
![image](https://github.com/hannibal002/SkyHanni/assets/24389977/c8802150-0bd2-4f25-b610-8133c69ab91b)